lib/paypal_express/api.rb

```diff
@@ -288,7 +295,8 @@ def get_payer_info(token,
         payment_processor_account_id = payment_processor_account_id || :default
         gateway                      = lookup_gateway(payment_processor_account_id, kb_tenant_id)
         gw_response                  = gateway.details_for(token)
-        response, transaction        = save_response_and_transaction(gw_response, :details_for, kb_account_id, kb_tenant_id, payment_processor_account_id, kb_payment_id, kb_payment_transaction_id, transaction_type)
+
+        response = @response_model.create_response(:details_for, kb_account_id, kb_payment_id, kb_payment_transaction_id, transaction_type, payment_processor_account_id, kb_tenant_id, gw_response)
```

Comment:
Do we need to save the `details_for` response with `payment` and `transaction` id? If not, then the changes at `get_payment_info` can be avoided. The `details_for`'s response won't be returned as part of the `get_payment_info`